tests/e2e/adminPage.spec.ts

```diff
@@ -0,0 +1,165 @@
+// adminPage.spec.ts
+import {
+  test,
+  expect,
+  chromium,
+  Browser,
+  BrowserContext,
+  Page
+} from "@playwright/test"
+import { AdminPage } from "./page_objects/adminPage"
+// import { authenticate } from './auth.utils'
+
+test.describe("Admin Page", () => {
+  let browser: Browser
+  let context: BrowserContext
+  let page: Page
+
+  test.beforeAll(async () => {
+    // Create a new browser instance
+    browser = await chromium.launch()
+    context = await browser.newContext() // Create a new context for all tests
+    page = await context.newPage()
+
+    const adminEmail =
+      process.env.TEST_ADMIN_USERNAME ?? "testadmin@example.com" // Emulator admin account
+
+    const adminPassword = process.env.TEST_ADMIN_PASSWORD ?? "password" // Emulator admin password
+
+    // Replace the credentials with valid admin credentials
+    await page.goto("http://localhost:3000")
```

Comment:
Does this need to point to the dev site (`maple-dev.vercel.app`) when we run this against dev? 

Looks like we may want to finally add the additional Playwright steps (as in https://playwright.dev/docs/ci#on-deployment) to supply the right url. This isn't necessary for this PR, but if not, we should create a ticket to add it in an immediate followup to ensure future playwright tests are setup for success.